modules/evs/fsw/src/cfe_evs_task.h

```diff
@@ -89,6 +90,9 @@ typedef struct
     uint8  ActiveFlag;           /* Application event service active flag */
     uint8  EventTypesActiveFlag; /* Application event types active flag */
     uint16 EventCount;           /* Application event counter */
+    CFE_TIME_SysTime_t LastSquelchReturnedTime; /* Time of last squelch token return */
```

Comment:
Recommend to use `OS_time_t` here (goes along other comment about using CFE_PSP_GetTime)